cobra/test/test_io/test_json.py

```diff
@@ -29,13 +29,12 @@ def test_load_json_model(data_directory, mini_model):
     assert compare_models(mini_model, json_model) is None
 
 
-@pytest.mark.xfail(reason="schema outdated")
 def test_save_json_model(tmpdir, mini_model):
     """Test the writing of JSON model."""
     jsonschema = pytest.importorskip("jsonschema")
     output_file = tmpdir.join("mini.json")
     cio.save_json_model(mini_model, output_file.strpath, pretty=True)
     # validate against JSONSchema
-    with open(output_file, "r") as infile:
+    with open(output_file.strpath, "r") as infile:
```

Comment:
The following seems more natural to me.

```suggestion
    with open(str(output_file), "r") as infile:
```